osprey-mock-service.js

```diff
@@ -48,6 +48,14 @@ function createServerFromBaseUri (raml, options) {
   var app = osprey.Router()
   var path = (raml.baseUri || '').replace(/^(\w+:)?\/\/[^/]+/, '') || '/'
 
+  if (options.definition) {
+    app.use('/resources', function (req, res) {
+      var body = '< link:"' +
+        options.definition +
+        '" rel="describedby" type="application/raml+yaml">'
```

Comment:
The `>` here is wrong too. This should follow the header `Link` syntax as defined [here](https://www.w3.org/TR/powder-dr/#httplink).